packages/x-tree-view/src/internals/models/treeView.ts

```diff
@@ -1,7 +1,7 @@
 import type { TreeViewAnyPluginSignature } from './plugin';
 import type { MergePluginsProperty } from './helpers';
 
-export interface TreeViewNode {
+export interface TreeViewItem {
```

Comment:
Similar to my remark on the variable returned by `getNode`, this one actually refers to the concept of "node" (that we might want to rename to "metadata" at some point), it's not the item itself.